home_page.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>global-talk-templat</title>
+    <script src="https://code.iconify.design/iconify-icon/1.0.3/iconify-icon.min.js"></script>
+</head>
+<body>
+    <header class="site-navigation">
+            <a class="menu-list-icon" href="menu_page.html">
+                <iconify-icon icon="bx:menu-alt-left"></iconify-icon>
+            </a>
+        <h1 class="logo-title">GlobalTalk</h1>
+        <div class="user-account">
+            <a class="user-account-link" href="">
+                <img class="avatar" src="user_avatar.svg" alt="user avatar">
+                <span class="user-name">Alex</span>
+                <span class="profile-list-icon"><iconify-icon icon="mdi:menu-down-outline"></iconify-icon></span>
+            </a>
+        </div>
+    </header>
+
+    <section class="ad-info">
+        <ul class="ad-info-list">
+            <li class="ad-info-block odd">
+                <div class="ad-info-block-text">
+                    <p class="meaning-platform">E-COURSE PLATFORM</p>
```

Comment:
Currently "Montserrat" font family is not applied to this paragraph as well as font-weight